appveyor.yml

```diff
@@ -31,7 +31,7 @@ build_script:
 before_test:
   - copy /y build\dec265\%CONFIGURATION%\dec265.exe build
   - copy /y build\enc265\%CONFIGURATION%\enc265.exe build
-  - copy /y build\libde265\%CONFIGURATION%\libde265.dll build
+  - copy /y build\libde265\%CONFIGURATION%\de265.dll build
```

Comment:
@farindk Is this an expected change?

Reply:
The old name in Windows was `libde265.dll`. I think we should keep the name and consider the change a bug.